AllReadyApp/Web-App/AllReady/wwwroot/css/site.css

```diff
@@ -138,15 +138,23 @@ Navigation
     color: #fff;
 }
 
-.navbar-default .navbar-nav > li:hover, .navbar-default .navbar-nav > li:active, .navbar-default .navbar-nav > li:focus {
+.navbar-default .navbar-nav > li:hover,
+.navbar-default .navbar-nav > li:active,
+.navbar-default .navbar-nav > li:focus {
     background-color: #ab6627;
 }
 
-.navbar-default .navbar-nav > li > a:hover, .navbar-default .navbar-nav > .active > a, .navbar-default .navbar-nav > .active > a:hover, .navbar-default .navbar-nav > .active > a:focus {
-    background: none;
-    color: #fff;

```

Comment:
The deletion of this code makes the root of the Admin menu black on hover even when it's not active. Despite the fact that Chrome thinks white is the prevailing color the `.navbar-default .navbar-nav > li > a:hover` selector from `navbar.less` does in fact have higher specificity (by 1) than `.navbar-default .navbar-nav > li > a` from `site.css`.
![image](https://cloud.githubusercontent.com/assets/775229/13209764/5eea1a8a-d8ed-11e5-9159-c063a1047c7f.png)